modules/test/playwright/pages/commerce/commerceChannelDefaultsPage.ts

```diff
@@ -109,5 +112,16 @@ export class CommerceChannelDefaultsPage {
 			exact: true,
 			name: 'Save',
 		});
+		this.channelEntryHeader = (channelEntryHeaderName: string) => {
+			return page.locator(`h3:has-text("${channelEntryHeaderName}")`);
+		};
+		this.channelEntry = (channelEntryName: string) => {
+			return page.locator(`[data-qa-id*='${channelEntryName}']`);
```

Comment:
try with `page.getByTestId(channelEntryName)` ---> I don't remember if works also with the "contains"

Reply:
does not seem to work for me:
"locator resolved to 0 elements"

Reply:
because from the doc seems that it supports also regex (https://playwright.dev/docs/api/class-page#page-get-by-test-id).
Can you try with something like this?
```
const xxx = new RegExp(`.*${variable}.*`, "g");
await page.getByTestId(xxx)....
```